setup.py

```diff
@@ -64,7 +64,7 @@
 
 base_dir = os.path.dirname(os.path.abspath(__file__))
 
-_date = '20240409'
+_date = '20240418'
 _libtpu_version = f'0.1.dev{_date}'
 _libtpu_storage_path = f'https://storage.googleapis.com/cloud-tpu-tpuvm-artifacts/wheels/libtpu-nightly/libtpu_nightly-{_libtpu_version}-py3-none-any.whl'
 _jax_version = f'0.4.27.dev{_date}'
```

Comment:
@alanwaketan how should I set the jax version?

Reply:
The pin update doc has the steps:
1. Occasionally, you will need to bump _jax_version like https://github.com/pytorch/xla/pull/6893, visit https://storage.googleapis.com/jax-releases/jax_nightly_releases.html to grep the jax version corresponding to the libtpu date.
2. Then test pip install torch_xla[pallas] -f https://storage.googleapis.com/jax-releases/jax_nightly_releases.html -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html to see if it actually install the correct version of jax and jaxlib.

Let me know if the above steps are not clear.

Reply:
0.4.27 works!